File: OCH_Tao/ABC/B/110.py
N,M,X,Y = map(int,input().split())
x = sorted(list(map(int,input().split())))
y = sorted(list(map(int,input().split())))
if max(x)<min(y) and max(x)>X and min(y)<Y:
  print("No War")
else:
  print("War")

File: OCH_Tao/ABC/B/110.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))
const [N, M, X, Y] = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
const x = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
const y = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
if (Math.max(...x) < Math.min(...y) && Math.max(...x) > X && Math.min(...y) < Y) {
  console.log("No War");
} else {
  console.log("War");
}